.buildkite/x-pack/elastic-agent/pipeline.xpack.elastic-agent.package.yml

```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
```

Comment:
If only for `7.17`, I'd suggest this file should not be committed in this PR, but create a new PR targeting `7.17` with this file.

Otherwise, if someone uses the pipeline with the default branch `main` the build will not fail but will print hello?